src/components/dashboard/purchase-card.svelte

```diff
@@ -3,36 +3,33 @@
   import Button from 'ui/button.svelte';
   import UnclickableChip from 'ui/unclickable-chip.svelte';
   import parseColor from '@/utils/optimal-color';
```

Comment:
`parseColor` doesn't really reflect what the function is doing. I imported it under the name `getBackground` elsewhere, maybe it's worth changing. And while you're at it, append a `.js` suffix to the import path for consistency.